FastSimulation/SimplifiedGeometryPropagator/python/TrackerMaterial_cfi.py

```diff
@@ -349,5 +365,26 @@
             ),
         )
     )
-    
-    
+
+phase1Pixel.toModify(TrackerMaterialBlock, TrackerMaterial = dict(
+        BarrelLayers = TrackerMaterialBlock.TrackerMaterial.BarrelLayers + [
+            #PIXB4                                                                                                                        
+            cms.PSet(
+                #radius = cms.untracked.double(16),                                                                                        
+                limits = cms.untracked.vdouble(0.0, 28.391),
+                thickness = cms.untracked.vdouble(0.0217),
+                activeLayer = cms.untracked.string("BPix4"),
+                interactionModels = _trackerMaterialInteractionModels
+                ),]
+        ))
+
+phase1Pixel.toModify(TrackerMaterialBlock, TrackerMaterial = dict(
+        EndcapLayers = TrackerMaterialBlock.TrackerMaterial.EndcapLayers + [
+            #PIXD3                                                                                                                 
+            cms.PSet(
+                limits = cms.untracked.vdouble(4.823, 16.598),
+                thickness = cms.untracked.vdouble(0.058),
+                activeLayer = cms.untracked.string("FPix3"),
+                interactionModels = _trackerMaterialInteractionModels
+                ),]
+        ))
```

Comment:
Please do a single call to `phase1Pixel.toModify()` along
```python
phase1Pixel.toModify(TrackerMaterialBlock, TrackerMaterial = dict(
    BarrelLayers = ...,
    EndcapLayers = ...
)
```

Also, now that you have enabled these customizations, you should remove the BPix4+FPix3 additions from the original definition of `TrackerMaterial`.

Reply:
Yes, did it.